account/account.go

```diff
@@ -14,6 +14,7 @@ import (
 
 	"github.com/bnb-chain/bsc-mev-sentry/log"
 	"github.com/bnb-chain/bsc-mev-sentry/node"
+	"github.com/bnb-chain/bsc-mev-sentry/syncutils"
 )
 
```

Comment:
I find this sentry does not have a decent readme, please draft a detailed reame, should including the introduction, framework, deployment suggestion, keymanagement.

Reply:
readme added